internal/converter/types.go

```diff
@@ -290,6 +291,8 @@ func (c *Converter) convertField(curPkg *ProtoPackage, desc *descriptor.FieldDes
 			jsonSchemaType.Items.Enum = jsonSchemaType.Enum
 			jsonSchemaType.Enum = nil
 			jsonSchemaType.Items.OneOf = nil
+			jsonSchemaType.Items.Ref = jsonSchemaType.Ref
+			jsonSchemaType.Ref = ""
```

Comment:
change here to show the ref nested inside items (same as how an actual object would be referenced)